examples/delete_projects.py

```diff
@@ -0,0 +1,42 @@
+'''
+Created on Nov 8, 2019
+
+@author: ylei
+
+Delete multiple projects and their scans. The project names to be deleted are kept in clean_project.json
+'''
+
+import argparse
+import csv
+import logging
+import sys
+import json
+import arrow
+
+from blackduck.HubRestApi import HubInstance
+
+parser = argparse.ArgumentParser("A program that will delete projects along with their scans")
+#parser.add_argument("project", help="Project name")
+parser.add_argument("-f", "--filename", default="clean_project.json", help="File to keep Project names")
```

Comment:
It looks like a filename is required so I would change this to make it an argument, as opposed to an option.

Reply:
Do you mean that making it as positional argument?
For my case I have set "clean_project.json" as default so even though user didn't specify a filename the script will still try to locate file named "clean_project.json".